psi4/versioner.py

```diff
@@ -293,6 +283,20 @@ def version_formatter(formatstring='{version}'):
         handle.write(main_fn)
 
 
+def write_new_cmake_metafile(versdata, outfile='metadata.out.cmake'):
+    main_fn = """
+include(CMakePackageConfigHelpers)
+
+write_basic_package_version_file(
+        ${{WTO}}/${{PN}}ConfigVersion.cmake
+        VERSION {ver}
+        COMPATIBILITY AnyNewerVersion)
+"""
+
+    with open(os.path.abspath(outfile), 'w') as handle:
+        handle.write(main_fn.format(ver=versdata['__version_cmake']))
```

Comment:
Pretty cool!

Reply:
Yep, cmake throws up all sorts of barriers if you try to do something it doesn't want to, but it does allow a few neat tricks, `cmake -P` and `cmake -C` included.